pkg/apis/noobaa/v1alpha1/noobaa_types.go

```diff
@@ -237,6 +244,20 @@ type AutoscalerSpec struct {
 	PrometheusNamespace string `json:"prometheusNamespace,omitempty"`
 }
 
+type BucketLoggingSpec struct {
+	// LoggingType specifies the type of logging for the bucket.
+	// There are two types available: guaranteed and non-guaranteed logging.
+	// - guaranteed logging ensures that all log entries are recorded without fail.
+	// - non-guaranteed logging may not record all log entries
+	// By default, guaranteed logging is enabled if no type is specified.
+	// +optional
+	LoggingType BucketLoggingTypes `json:"loggingType,omitempty"`
+
+	// TODO
+	// +optional
+	BucketClass string `json:"bucketclass,omitempty"`
```

Comment:
I guess that's should be storageclass right? it's the type of storageclass that we will use to create a PVC on top of in case of using guaranteed logging.